packages/net/tcpconnection.pony

```diff
@@ -138,6 +149,17 @@ actor TCPConnection
     @pony_os_peername[Bool](_fd, ip)
     ip
 
+  fun ref expect(qty: USize = 0) =>
+    """
+    A `received` call on the notifier must contain exactly `qty` bytes. If
+    `qty` is zero, the call can contain any amount of data. This has no effect
+    if called in the `sent` notifier callback.
```

Comment:
It's not clear (to me) from this docstring that any data in excess of the `qty` bytes will be simply truncated and thrown away, but looking at the implementation that seems to be what it is doing.

It seems important to know that data will be thrown away.  Do you think this is worthy of more explanation in the docstring, or am I just not getting the current docstring well enough?